packages/plugins/opentelemetry/src/api.ts

```diff
@@ -0,0 +1,63 @@
+import { Logger } from '@graphql-hive/logger';
+import { context, Tracer } from '@opentelemetry/api';
+import type { OpenTelemetryPluginUtils } from './plugin';
+
+export * from '@opentelemetry/api';
+
+export type HiveAPI = Omit<OpenTelemetryPluginUtils, 'tracer'> & {
+  tracer?: Tracer;
```

Comment:
You did great with commenting the rest of options, lets leave comment for this one too. What is it? When is it undefined?